python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -153,9 +153,15 @@ cpdef generate_pandas_metadata(Table table, index):
         types,
     )
 
-    md = metadata[b'pandas']
-    json_str = md.decode("utf-8")
-    return json_str
+    md_dict = json.loads(metadata[b"pandas"])
```

Comment:
Can we just add the try / except of `import ujson as json` since it cuts the time in half of doing this?

Reply:
👍 done